pkgs/development/python-modules/flask/default.nix

```diff
@@ -26,6 +27,7 @@ buildPythonPackage rec {
     itsdangerous
     jinja2
     werkzeug
+    setuptools
```

Comment:
Might be worth including a comment here if it isn't a dependency in setup.py to make it clear why this is here, or it might end up getting removed by a well-meaning maintainer in future.

Reply:
Agreed.

Reply:
Done.

Reply:
even better would be to report it to flask upstream. Most package authors don't realize that having setuptools as build system doesn't automatically mean it will be available during runtime.

Reply:
Reported.